### DISCUSSION TITLE
Add QMK support for Boardsource Unicorne

### DISCUSSION THREAD

**Discussion by yooneo** [manna-harbour/miryoku#270] [Q&A] [answered]
Please add keyboard support for Boardsource Unicorne. 

Ref: https://github.com/qmk/qmk_firmware/tree/master/keyboards/boardsource/unicorne

**Comment by manna-harbour** [comment, marked as answer]
`boardsource/unicorne` uses the split_3x6_3 community layout so it already supported by Miryoku QMK, see https://github.com/manna-harbour/miryoku_qmk/tree/miryoku/users/manna-harbour_miryoku#split_3x6_3.

Unicorne support in QMK was added since the `miryoku` branch was last updated. Ordinarily you could build directly from, or by merging with, QMK master. Unfortunately, that's no longer possible, see https://github.com/manna-harbour/miryoku/discussions/287.

However, there is a workaround described in that discussion. Here's a sample build run: https://github.com/manna-harbour/miryoku_qmk/actions/runs/7515355552.
